GiniComponents/GiniUtilites/Sources/GiniUtilites/Utilites.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Utilites.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import Foundation
+
+/**
+ Returns a decimal value
+
+ - parameter inputFieldString: String from input field.
+
+ - returns: decimal value in current locale.
+ */
+
+public func decimal(from inputFieldString: String) -> Decimal? {
```

Comment:
Suggestion: Maybe we should add this to a String extension into our SDK?!

Reply:
done